master/docs/manual/cfg-www.rst

```diff
@@ -571,12 +574,9 @@ Here is a configuration that is known to work (Apache 2.4.10 / Debian 8), direct
         ServerName buildbot.example
         ServerAdmin webmaster@buildbot.example
 
-        <Location /ws>
-          ProxyPass ws://127.0.0.1:8020/ws
-          ProxyPassReverse ws://127.0.0.1:8020/ws
-        </Location>
-
-        ProxyPass /ws !
+        # replace with actual port of your Buildbot master
+        ProxyPass ws://127.0.0.1:8020/ws
```

Comment:
@Frodox, sure here it is

```
<VirtualHost *:443>
        ServerName bb9.example
        ServerAdmin webmaster@bb9.example
        <Location />
          AuthType Basic
          AuthBasicProvider ldap 
          AuthName "Please login"
          AuthLDAPUrl "ldaps://ldap1.example ldap2.example/dc=example?uid?sub?(objectClass=*)" NONE

          AuthLDAPBindDN cn=apache,dc=example
          AuthLDAPBindPassword SECRET

          Require valid-user

          RewriteEngine On
          RewriteCond %{LA-U:REMOTE_USER} (.+)$
          RewriteRule . - [E=RU:%1,NS]
          RequestHeader set REMOTE_USER %{RU}e
        </Location>

        ProxyPass /ws ws://localhost:8011/ws
        ProxyPassReverse /ws ws://localhost:8011/ws
        ProxyPass / http://localhost:8011/
        ProxyPassReverse / http://localhost:8011/

        SetEnvIf X-Url-Scheme https HTTPS=1
        ProxyPreserveHost On


       SSLEngine on
       # more SSL stuff

</VirtualHost>
```

As you can see, I'm doing basic auth on LDAP (don't think it should interfere too much).

This is a Debian 8  system, with the standard current apache2 package, and the default MPM config.

```
$ apt-cache policy apache2
apache2:
  Installed: 2.4.10-10+deb8u8
  Candidate: 2.4.10-10+deb8u8
  Version table:
 *** 2.4.10-10+deb8u8 0
        500 http://ftp.debian.org/debian/ jessie/main amd64 Packages
        500 http://security.debian.org/ jessie/updates/main amd64 Packages
        100 /var/lib/dpkg/status
```

As you may know, Debian's configuration is split in several files. I don't have anything special besides the virtualhost. Notably, /etc/apache2/mods-enabled/proxy.conf is the stock one.

A side effect is always possible, but it doesn't seem very likely.

Hope this helps.